manifests/z/Zwift/Zwift/1.0.52/Zwift.Zwift.installer.yaml

```
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Zwift.Zwift
+PackageVersion: 1.0.52
+MinimumOSVersion: 10.0.0.0
+InstallerSuccessCodes:
+- 4294967295
+InstallModes:
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://cdn.zwift.com/app/ZwiftSetup.exe
+  InstallerSha256: 0FA151B1ADF8813E93EFFD8154CBF3E8437D4E7D78E8AAE29DFAAF451B2D2028
+  InstallerSwitches:
+    Silent: /VERYSILENT /NORESTART
+    SilentWithProgress: /SILENT /NORESTART
```

Where did you get the user scope from?

```suggestion
  Architecture: x64
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://cdn.zwift.com/app/ZwiftSetup.exe
  InstallerSha256: 0FA151B1ADF8813E93EFFD8154CBF3E8437D4E7D78E8AAE29DFAAF451B2D2028
```